### PR TITLE
Updated script

### PR DESCRIPTION
**Pull request by megan-oncoshot**

### CODE REVIEW

**Comment by megan-oncoshot** [comment]
@sergei530 @ak-oncoshot please review this script which will be used to delete imported profiles

File: src/delete.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+        // Write Log to file
+        fs.writeFile(`./logs/${Date.now()}-delete-log.txt`, log, "utf8", (res, err) => {
+            if (err) {
+                console.log(err)
+            }
+        })
+    })
+}
+console.log("This tool deletes the formatted JSON file to the Oncoshot API.\n")
```
Comment (ak-oncoshot) [inline src/delete.js:120]: Description not clear. I believe it deletes "profiles with ids from provided JSON file" but not the file

File: src/delete.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+console.log("Guide to statusCodes:")
+console.log("204: Success, updated existing profile")
```
Comment (ak-oncoshot) [inline src/delete.js:122]: 204: Success, removed existing profile

**Review by ak-oncoshot — APPROVED** [review]

**Review by sergei530 — APPROVED** [review]

File: src/delete.js (hunk 1 above)
Comment (megan-oncoshot) [reply src/delete.js:120]: Ok. Fixed.

File: src/delete.js (hunk 2 above)
Comment (megan-oncoshot) [reply src/delete.js:122]: Ok. Fixed!

**Pull request merged**